### ISSUE TITLE
Can you release the prototxt of se-mobilenet?

### ISSUE THREAD

**Issue by Maobiao007** [hujie-frank/SENet#64]
Have tried to train from the scratch, but did not get the accuracy noted in the paper.
Maybe the structure is a little tricky?

**Comment by hujie-frank** [comment]
[SE-MobileNet.txt](https://github.com/hujie-frank/SENet/files/2915440/SE-MobileNet.txt)
Here, we use  batchnorm layer implemented by CuDNN.

**Comment by liuxuepdf** [comment]
Hello, the SE-mobilenet.txt you uploaded may have some mistakes, the SE block did not concat to the scale layer, and there no "reshape" layer in your txt. 

layer {
  name: "scale6"
  type: "Scale"
  bottom: "conv6/pw"
  bottom: "reshape6"
  top: "conv6"
  scale_param {
    axis: 0
    bias_term: false
  }
}

**Comment by hujie-frank** [comment]
[SE-MobileNet.txt](https://github.com/hujie-frank/SENet/files/2918114/SE-MobileNet.txt)
Sorry for this mistake,  I have corrected it.

**Comment by liuxuepdf** [comment]
> [SE-MobileNet.txt](https://github.com/hujie-frank/SENet/files/2918114/SE-MobileNet.txt)
> Sorry for this mistake, I have corrected it.

Thanks for your reply!
